docs/updating-dependencies.md

```diff
@@ -9,37 +12,23 @@ To update golang dependencies one has to choose between two options:
 ## Update the whole dependency tree
 
 Done when the need to catch upstream dependencies arises, it can be done by
-calling
-
-```bash
-make vendor
```

Comment:
Should the makefile target be deprecated / removed altogether? I'm curious on if we still need it for anything if we're removing it from docs.

Reply:
I was asking myself the same question. I see Ben updated the script a few weeks ago [PR #3301](https://github.com/Azure/ARO-RP/pull/3301)  

He said it still has some uses, so I don't think we should remove it. Documentation on it's specific purposes and uses would be nice.